scripts/generateAbi.js

```diff
@@ -1,19 +1,26 @@
 const fs = require("fs");
 const solc = require("solc");
 
+// Assuming each contract is in the file with the same name.
```

Comment:
Can we do a simple file search for all top-level `*.sol` files instead?

Reply:
There is a file `PythStructs.sol` that is a contract with no method and its abi is empty (actually I expected it to include all structs but it didn't!). I want to avoid having that published.